glossary.md

```diff
@@ -13,52 +13,73 @@ gh: specifically for github
 
 ### Staging
 
-Github uses staging, which means marking changes in the working directory to be included in your next commit. Basically, before saving your progress, you can choose what changes you want to include. 
+Git uses staging, which means marking changes in the working directory to be included in your next commit. Basically, before saving your progress, you can choose what changes you want to include. 
 
+Let's use an analogy to simplify this. Let's say we have a group project and everyone has their own notebooks with their own research. To make it easier, we decide to make a shared binder where everyone can share their notes. 
+
+*Staging Area*: Temporary placeholder for your files that you plan to add and review. Think of it as a folder within your binder where you temporarily place the papers (files) you're ready to add to your project. This folder helps you organize and review what will be added to the binder.
+
+The shared binder is called a **REMOTE REPOSITORY.** It allows users to add their code in from their local computers to the shared repository on cloud.
 
 
 ## Terminal Commands for Github
 
 ### Setting up & Initalizing
-| command    | use |
-| -------- | ------- |
-| git config --global user.name "[firstname lastname]" | Sets up a name that can be identifible    |
-| git config --global user.email "[email]" | sets up the email address/account that is associated with changes     |
+| command    | use | 
+| ------- | ------- |
+| git config --global user.name "[firstname lastname]" | Sets up the name that you use whenever you commit a change to a repository. If you save your changes, it will be under the name you put in so when people refer they can see its YOUR changes.    |
+| git config --global user.email "[email]" | sets up the email address/account that is associated with your changes in the repository    |
 | git init    | initalize an existing directory as Git Repository    |
-| git clone [url] <br> gh repo clone [Organization/Repository] | clone the repository from cloud to your local host. Changes locally will hav to be pushed to cloud to update the repo |
+| git clone [url] <br> gh repo clone [Organization/Repository] | clone the repository from cloud to your local host. Changes locally will have to be pushed to cloud to update the repo |
```

Comment:
maybe explain cloning a bit more? like its downloading the code off the internet AND downloading all of the git repo metadata and change history